src/release/main.d

```diff
@@ -679,9 +684,14 @@ ActionList clearReleaseNotes ( )
 {
     ActionList list;
 
-    list.actions ~= new LocalAction("git rm relnotes/*.md",
+    import std.file: dirEntries, SpanMode;
+    import std.array: array;
+    import std.algorithm.iteration: map;
+    list.actions ~= new LocalAction(["git", "rm"] ~ array(dirEntries("relnotes", "*.md",
+                                     SpanMode.shallow).map!("a.name")),
```

Comment:
I'd suggest non-string lambda and some pipeline styling :P 

```D
auto relnotes = dirEntries("relnotes", "*.md", SpanMode.shallow)
       .map!(a => a.name)
       .array();
list.actions ~= new LocalAction(["git", "rm"] ~ relnotes,  "Removing release notes");
```

Reply:
I tried the chaining at first but it didn't work, it said something like "MapResultWhatever doesn't have a method array". I could change the `map`, but what would be the advantage?